dist/angular-sticky-table-header.js

```diff
@@ -109,7 +109,8 @@ angular.module('turn/stickyTableHeader', ['watchDom']).value('stickyTableHeaderO
             } else if (scope.stuck && scrollY < scope.offset.top) {
               scope.setStuck(false);
             }
-            scope.clone.css('left', scope.offset.left - scrollX);
```

Comment:
@bcherny . Ok. We have removed code which update the left position in `setCloneGutter`.
Set `left:0` in css instead of having to re-set it on every tick.